vspace/src/main/java/edu/asu/diging/vspace/web/staff/SpaceController.java

```diff
@@ -83,4 +89,17 @@ public ResponseEntity<Map<String,Object>> showSpaceLinks(@PathVariable String id
         return new ResponseEntity<>(responseData, HttpStatus.OK);
     }
 
+    @RequestMapping(value = "/staff/space/order/mode", method = RequestMethod.POST)
+    public String updateSpaceOrderMode(String mode, @RequestParam String spaceOrderMode) {
+        if(spaceOrderMode.equalsIgnoreCase("ALPHABETICAL")) {
```

Comment:
You can get an `ExhibitionSpaceOrderMode` from the enum via its string representation and get rid of the if statement.